pkg/apps/application-list.jsx

```diff
@@ -140,18 +140,20 @@ class ApplicationList extends React.Component {
             tbody = <tr className="app-list-empty"><td>{empty_caption}</td></tr>;
         } else {
             table_classes += " table-hover";
-            tbody = comps.map((c) => <ApplicationRow comp={c} />);
+            tbody = comps.map((c) => <ApplicationRow comp={c} key={c} />);
```

Comment:
`c` is not a string.  `c.id` should work.

Reply:
ok